app/models/iiif_content_search_response.rb

```diff
@@ -108,6 +118,18 @@ def text
       highlight[first...last].gsub(%r{</?em>}, '')
     end
 
+    def before
+      first = highlight.index('<em>')
+      highlight[0...first].split.map { |x| split_word_and_payload(x) }.map(&:first).join(' ')
+    end
+
+    def after
+      last = highlight.rindex('</em>')
+      highlight[(last + 6)..highlight.length].split.map { |x| split_word_and_payload(x) }.map(&:first).join(' ')
```

Comment:
Is this 6 because a space will always follow the highlight (or when it doesn't we're at the end of the field)?

I would think we would want to add 5 to the index since that's the number of characters in `</em>`, so this made me pause for a moment.